build.gradle

```diff
@@ -16,8 +16,121 @@
  * specific language governing permissions and limitations
  * under the License.
  */
+// TODO: this is work in progress, please follow FINERACT-1171
 buildscript {
+    ext {
+        jacocoVersion = '0.8.5'
+    }
+    repositories {
+        jcenter()
+        maven { url "https://plugins.gradle.org/m2/" }
+    }
+
+    dependencies {
+        classpath 'com.bmuschko:gradle-cargo-plugin:2.7.1'
+        classpath 'org.zeroturnaround:gradle-jrebel-plugin:1.1.10'
+        classpath 'org.springframework.boot:spring-boot-gradle-plugin:2.3.4.RELEASE'
+        classpath 'gradle.plugin.com.hierynomus.gradle.plugins:license-gradle-plugin:0.15.0'
+        classpath 'org.apache.openjpa:openjpa:3.1.2' // when upgrading, also change OpenJPA version repeated below in dependencyManagement!
+        classpath 'com.radcortez.gradle:openjpa-gradle-plugin:3.1.0'
+        classpath 'org.nosphere.apache:creadur-rat-gradle:0.7.0'
+        classpath "gradle.plugin.com.github.spotbugs.snom:spotbugs-gradle-plugin:4.5.1"
+        classpath "gradle.plugin.com.github.andygoossens:gradle-modernizer-plugin:1.3.0"
+        classpath "gradle.plugin.com.gorylenko.gradle-git-properties:gradle-git-properties:2.2.3"
+        classpath "net.ltgt.gradle:gradle-errorprone-plugin:1.2.1"
+        classpath "io.swagger.core.v3:swagger-gradle-plugin:2.1.5"
+        classpath "gradle.plugin.org.hidetake:gradle-swagger-generator-plugin:2.18.2"
+    }
+}
+
+plugins {
+    id 'io.spring.dependency-management' version '1.0.10.RELEASE'
+    id 'com.diffplug.spotless' version '5.6.1'
+}
+
+description = '''\
+Run as:
+gradle clean bootRun'''
+
+project.ext.jerseyVersion = '1.19.4'
+ext['groovy.version'] = '3.0.6'
+
+allprojects  {
+    group = 'org.apache.fineract'
+    version = "$releaseVersion"
```

Comment:
This line is resulting in regression of FINERACT-1187... i.e. when you define a property "version" then war task will automatically append the "version" to the WAR name